software/firmware/europi.py

```diff
@@ -28,6 +28,7 @@
 from version import __version__
 
 from framebuf import FrameBuffer, MONO_HLSB
+from largefont_writer import Writer

```

Comment:
I agree. I didn't how you would like to integrate this large_font support. 

Today, I'm not sure that including the large_font support in the base firmware is a good thing. Font files take up quite a bit of memory and it's really more a nice-to-have feature than a must-have.

If we still want to integrate the support in the base firmware, then the largefont_writer could be directly written in the europy module, like the other classes (Display, Button, ...). But, the fonts themselves should remain outside the firmware because they are large and the user may want to use its own custom fonts.

Reply:
I also agree that the firmware shouldn't have dependencies from experimental. I think you could solve this by creating a new Display class in the experimental folder with europi.Display as a base class and add all of these new methods. Then script authors could override the global `oled` variable with a new Experimental Display instance in our scripts.